code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,72 @@
+package challenges.linkedlist;
+
+import java.util.ArrayList;
+
+public class LinkedList {
+   public  Node head;
+   public int length;
+
+    public void LinkedList(){
+            this.head = null;
+            this.length = 0;
+    }
+    public void insert(int data){
+        try
+        {
+            Node node = new Node(data);
+            node.next = null;
+            if(this.head == null){
+                this.head = node;
+                this.length++;
+            } else {
+                Node current = this.head;
+                node.next = current;
+                this.head = node;
+                this.length++;
+            }
+
+        }
+        catch(IllegalArgumentException ex){
+            System.out.println("You either didn't pass anything in or you didn't pass in an Integer");
+        }
+
+
+    }
+
+    public boolean includes(int data){
+
+        Node current = this.head;
+        boolean isIncluded = false;
+        try
+        {
+            while(current != null){
+                if(current.data == data){
+                    isIncluded = true;
+                }
+                current = current.next;
+            }
+        }
+        catch (IllegalArgumentException ex)
+        {
+            System.out.println("You either didn't pass anything in or you didn't pass in an Integer");
+        }
+        return isIncluded;
+    }
+
+    public ArrayList print(){
```

Comment:
It's good practice to specify the data type an ArrayList will have, in this case:
`ArrayList<Integer>`